### PR TITLE
(SURR): Correct postprocess

### PR DESCRIPTION
**Pull request by ujpv**
Fix syntax error, which appears immediately after surrounder use.

### CODE REVIEW

**Comment by matklad** [comment]
Nice! Let's add a test here, to make sure this does not regress and to make it clear in the git history what is the purpose of these changes!

The test should look like this:

```
fun testCorrectPostProcess() = doTest(
    """
    fn main() {
        true.if<caret>
        1;
    }
    """
    ,
    """
    fn main() {
        if true {}
        1;
    }
    """
)
```

To check for the absence of errors,  you'll need to modify

```
    protected fun doTest(@Language("Rust") before: String, @Language("Rust") after: String) {
        myFixture.configureByText(RustFileType, before)
        checkApplicability(before, true)
        myFixture.type('\t')
        myFixture.checkHighlighting(false, false, false) // add this line
        myFixture.checkResult(after)
    }
```

**Comment by ujpv** [comment]
I added tests.
But it is a bit more complicated. Not always code after action is syntactically correct.
For example 

```Rust
    fn main() {
        <selection>println!("Hellio");<selection>
    }
```
Surround with while {} gives
```Rust
    fn main() {
        while <caret> {println!("Hellio");}
    }
```

**Comment by matklad** [comment]
Indeed! But in most of the cases it is correct? Then perhaps we should use ` checkSyntaxErrors = true` as the default?

**Comment by ujpv** [comment]
Of course.

**Comment by ujpv** [comment]
Done.

File: src/test/kotlin/org/rust/ide/template/postfix/IfPostfixTemplateTest.kt [hunk 1]
```diff
@@ -110,6 +110,8 @@ class IfPostfixTemplateTest : PostfixTemplateTestCase(IfExpressionPostfixTemplat
                 }
             }
             """
+            ,
+            checkSyntaxErrors = false
```
Comment (matklad) [inline src/test/kotlin/org/rust/ide/template/postfix/IfPostfixTemplateTest.kt]: This should actually be `true` :) There is a `;` missing after let. The test should be

```
             fn main() {
                 let a = if (true) {
                     true == false.if<caret>
                 } else {
                     false == true
                  }; // <-
              }
```

**Comment by ujpv** [comment]
Yes my fault.
Fixed.

**Comment by matklad** [comment]
CI fails...

**Comment by matklad** [comment]
Great!

**Pull request merged**